.buildkite/pipeline.yml

```diff
@@ -79,6 +79,8 @@
       python/ray/new_dashboard/...
     - bazel test --config=ci $(./scripts/bazel_export_options)
       python/ray/serve/...
+    - npm install cypress -g
```

Comment:
seems to be failed with 
```

npm WARN deprecated har-validator@5.1.5: this library is no longer supported
--
  | /usr/bin/cypress -> /usr/lib/node_modules/cypress/bin/cypress
  |  
  | > cypress@6.5.0 postinstall /usr/lib/node_modules/cypress
  | > node index.js --exec install
  |  
  | Cypress cannot write to the cache directory due to file permissions
  |  
  | See discussion and possible solutions at
  | https://github.com/cypress-io/cypress/issues/1281
  |  
  | ----------
  |  
  | Failed to access /root/.cache/Cypress:
  |  
  | EACCES: permission denied, mkdir '/root/.cache/Cypress'
  |  
  | ----------
  |  
  | Platform: linux (Ubuntu - 20.04)
  | Cypress Version: 6.5.0
  | npm ERR! code ELIFECYCLE
  | npm ERR! errno 1
  | npm ERR! cypress@6.5.0 postinstall: `node index.js --exec install`
  | npm ERR! Exit status 1
  | npm ERR!
  | npm ERR! Failed at the cypress@6.5.0 postinstall script.
  | npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
  |  
  | npm ERR! A complete log of this run can be found in:
  | npm ERR!     /root/.npm/_logs/2021-02-22T20_00_35_381Z-debug.log


```

let's try `sudo npm install` or just drop the `-g`, not sure which will work.

Reply:
changed to `sudo npm install`